Strings/CardName.jsonc

```
@@ -58,55 +58,55 @@
 
 	"IruCa（IruCa定期券）": {
 		"ja": "IruCa（IruCa定期券）",
-		"en": "IruCa（IruCa定期券）",
+		"en": "IruCa (Commuter Pass)",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"IruCa（キッズIruCa）": {
 		"ja": "IruCa（キッズIruCa）",
-		"en": "IruCa（キッズIruCa）",
+		"en": "IruCa (Kids)",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"IruCa（グリーンIruCa）": {
 		"ja": "IruCa（グリーンIruCa）",
-		"en": "IruCa（グリーンIruCa）",
+		"en": "IruCa (Green)",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"IruCa（ゴールドIruCa）": {
 		"ja": "IruCa（ゴールドIruCa）",
-		"en": "IruCa（ゴールドIruCa）",
+		"en": "IruCa (Gold)",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"IruCa（シニアIruCa）": {
 		"ja": "IruCa（シニアIruCa）",
-		"en": "IruCa（シニアIruCa）",
+		"en": "IruCa (Senior)",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"IruCa（スクールIruCa）": {
 		"ja": "IruCa（スクールIruCa）",
-		"en": "IruCa（スクールIruCa）",
+		"en": "IruCa (School)",
 		"zh-Hans": null,
 		"zh-Hant": null,
 		"zh-HK": null
 	},
 
 	"IruCa（フリーIruCa）": {
 		"ja": "IruCa（フリーIruCa）",
-		"en": "IruCa（フリーIruCa）",
+		"en": "IruCa (Free/Unregistered)",
```

IruCa の提供元である高松琴平電気鉄道の表記に合わせたいと思います。
I'd like to match the notation of Takamatsu Kotohira Electric Railway, the issuer of IruCa.

```suggestion
		"en": "IruCa (Free)",
```